test/e2e-api/admin/members.test.js

```diff
@@ -1365,7 +1365,7 @@ describe('Members API: with multiple newsletters', function () {
     });
 
     // Edit a member
-    it('Can and and edit with custom newsletters', async function () {
+    it('Can add and edit with custom newsletters', async function () {
```

Comment:
I believe this rename means we have redundant snapshots: https://github.com/TryGhost/Ghost/blob/c9a858d977be39ac71ebda2e489ab96ce772b596/test/e2e-api/admin/__snapshots__/members.test.js.snap#L2483

Reply:
🤔 Not sure why they didn't go away with `UPDATE_SNAPSHOTS=1`, i'll retry and see if it maybe just needs deletion of snapshot and recreate.

Reply:
Yeah delete and update worked to get rid of dead tests, its now updated on PR :)